source/_docs/guides/front-end-performance.md

```diff
@@ -0,0 +1,457 @@
+---
+title: Front End Performance
+description: Learn how to ace an online speed test with Pantheon's Global CDN.
+tags: [siteintegrations]
+categories: []
+type: guide
+image: CDN-speedTest-docs-guide
+permalink: docs/guides/:basename/
+contributors:
+  - tessak22
+  - stevector
+  - ataylorme
+  - rachelwhitton
+  - dwayne
+  - dgorton
+  - dts
+---
+Nobody likes a slow site. In this guide we’ll explore how to speed up front end performance to ensure that visitors to your site stay engaged and have the best experience.  
+
+<div class="enablement">
+ <a href="https://pantheon.io/agencies/learn-pantheon?docs"><h4 class="info">Agency DevOps Training</h4></a>
+ <p>Learn industry best practices for caching, how to take advantage of them on the platform, and troubleshooting common issues with help from the experts at Pantheon.</p>
+</div>
+
+
+There are many different ways to measure page speed and performance. This guide will be tuning for the metrics set forth by [Google's mobile speed test](https://testmysite.thinkwithgoogle.com/){.external}:
+
+- Reduce Server Response Time
+- Compress Images
+- Deliver Efficient CSS and JavaScript
+- Avoid Redirects
+- Leverage Browser Caching
+
+![Acing Google's speed test](/source/docs/assets/images/guides/front-end-performance/ace-it.png)
+
+## Quick Response Time
+<dt>Waiting</dt>
```

Comment:
This didn't read for me. Why is a headline that says "Quick Response Time" followed by a subhead that says "waiting"? Feels like there's a missing intro paragraph perhaps?

Reply:
Added this to explore opening sections with a definitions list, and I wasn't sure how I felt about it. Main goal is to define TTFB and TTFP for the next paragraph. 

I'll rewrite into paragraph form, it does feel a bit weird.